src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.cs

```diff
@@ -715,21 +738,32 @@ private static bool IsPathRootedSafe(string path)
             return path.IndexOfAny(Path.GetInvalidPathChars()) == -1 && Path.IsPathRooted(path);
         }
 
-        private void SetHttpWarningVisibilityForSelectSource(PackageSourceContextInfo selectedSource)
+        private void SetHttpErrorVisibilityForSelectSource(PackageSourceContextInfo selectedSource)
```

Comment:
Typo in method name.

```suggestion
        private void SetHttpErrorVisibilityForSelectedSource(PackageSourceContextInfo selectedSource)
```